### ISSUE TITLE
[a11y] Expandable Card - Hidden Items Can Be Tabbed Through

### ISSUE THREAD

**Issue by anechol** [Kajabi/sage-lib#876]
## Environment

> Operating system

MacOS

> Browser

- Firefox Developer Edition
- Google Chrome
- Microsoft Edge (Chromium)

> Testing tool used

Nerdefocus

> Assistive technology used to verify

N/A

## Detailed description

> What did you expect to happen?

Hidden content shouldn't be tabbed through when not visible.

> What happened instead?

Hidden items are able to be focused through. It's unclear where focus is after tabbing the Expandable Card button.

> What WCAG 2.1 success criteria does the issue violate?
[2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Steps to reproduce the issue

1. Navigate to Expandable Card in Dage docs
2. Tab through the example cards.

## Additional Information

https://user-images.githubusercontent.com/14791307/136860077-faf466cb-db62-4b13-a382-ffa083f82c49.mov
